hack/update-codegen.sh

```diff
@@ -36,58 +36,63 @@ group "Kubernetes Codegen"
 
-${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
+"${CODEGEN_PKG}"/generate-groups.sh "deepcopy,client,informer,lister" \
   knative.dev/eventing-kafka-broker/control-plane/pkg/client/internals/kafka knative.dev/eventing-kafka-broker/control-plane/pkg/apis/internals/kafka \
   "eventing:v1alpha1" \
-  --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
+  --go-header-file "${REPO_ROOT_DIR}"/hack/boilerplate/boilerplate.go.txt
 
-${CODEGEN_PKG}/generate-groups.sh "client,informer,lister" \
+"${CODEGEN_PKG}"/generate-groups.sh "client,informer,lister" \
   knative.dev/eventing-kafka-broker/third_party/pkg/client knative.dev/eventing-kafka-broker/third_party/pkg/apis \
   "keda:v1alpha1" \
-  --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
+  --go-header-file "${REPO_ROOT_DIR}"/hack/boilerplate/boilerplate.go.txt
 
 group "Knative Codegen"
 
 # Knative Injection
-${KNATIVE_CODEGEN_PKG}/hack/generate-knative.sh "injection" \
+"${KNATIVE_CODEGEN_PKG}"/hack/generate-knative.sh "injection" \
   knative.dev/eventing-kafka-broker/control-plane/pkg/client knative.dev/eventing-kafka-broker/control-plane/pkg/apis \
   "eventing:v1alpha1 messaging:v1beta1 sources:v1beta1 bindings:v1beta1" \
-  --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
+  --go-header-file "${REPO_ROOT_DIR}"/hack/boilerplate/boilerplate.go.txt
 
-${KNATIVE_CODEGEN_PKG}/hack/generate-knative.sh "injection" \
+"${KNATIVE_CODEGEN_PKG}"/hack/generate-knative.sh "injection" \
   knative.dev/eventing-kafka-broker/control-plane/pkg/client/internals/kafka knative.dev/eventing-kafka-broker/control-plane/pkg/apis/internals/kafka \
   "eventing:v1alpha1" \
-  --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
+  --go-header-file "${REPO_ROOT_DIR}"/hack/boilerplate/boilerplate.go.txt
 
-${KNATIVE_CODEGEN_PKG}/hack/generate-knative.sh "injection" \
+"${KNATIVE_CODEGEN_PKG}"/hack/generate-knative.sh "injection" \
   knative.dev/eventing-kafka-broker/third_party/pkg/client knative.dev/eventing-kafka-broker/third_party/pkg/apis \
   "keda:v1alpha1" \
-  --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
+  --go-header-file "${REPO_ROOT_DIR}"/hack/boilerplate/boilerplate.go.txt
 
 group "Update deps post-codegen"
 
 # Our GH Actions env doesn't have protoc, nor Java.
 # For more details: https://github.com/knative-sandbox/eventing-kafka-broker/pull/847#issuecomment-828562570
 # Also: https://github.com/knative-sandbox/knobots/runs/2609020026?check_suite_focus=true#step:6:291
 if ! ${GITHUB_ACTIONS:-false}; then
-  ${REPO_ROOT_DIR}/hack/generate-proto.sh
+  "${REPO_ROOT_DIR}"/hack/generate-proto.sh
 
   # Update Java third party file
   pushd data-plane
   ./mvnw -Dlicense.outputDirectory=. license:aggregate-add-third-party
   popd
+
+  # Run maven command to apply spotless formatting
+  pushd "${REPO_ROOT_DIR}/data-plane"
+  mvn spotless:apply
+  popd
```

Comment:
nit: why entering the dir and existing it and then entering it again to then exit it?